src/gatsby/models/github/index.js

```diff
@@ -7,34 +7,6 @@
    alone does.
  */
 
-async function createStaticGithubDataNode(api, fieldData = {}) {
```

Comment:
is it unused now?

Reply:
Yeah, it gets picked up by `lint-ts`.

Reply:
hmm ... why didn't it pick it up earlier?